virttest/qemu_devices/qcontainer.py

```diff
@@ -1359,9 +1362,17 @@ def machine_other(cmd=False):
 
         if params.get("vm_pci_hole64_fix"):
             if machine_type.startswith('pc'):
-                devices.append(qdevices.QGlobal("i440FX-pcihost", "x-pci-hole64-fix", "off"))
+                devices.append(
+                    qdevices.QGlobal(
+                        "i440FX-pcihost",
+                        "x-pci-hole64-fix",
+                        "off"))
```

Comment:
This is not really the favoured style. Something like:

```
                devices.append(qdevices.QGlobal("i440FX-pcihost",
                                                "x-pci-hole64-fix", "off"))
```

is much more condense and readable. Similar issues are everywhere (autopep is not really that good as human)

Reply:
OK,  let me revert the commit for fix code style in next version.